__tests__/app.test.js

```diff
@@ -110,3 +111,26 @@ describe('GET /api', () => {
         })
     })
 })
+
+describe('GET /api/articles/:article_id', () => {
+  it('responds with an article object with correct article_id given', () => {
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then((res) => {
+       
+        expect(Object.values(res.body.article).length).toBe(8)
+        expect(res.body.article.article_id).toBe(1)
+    })
+  })
+
+  it('responds with an error when the given valid id is not registered', () => {
+    return request(app)
+    .get('/api/articles/87964')
+    .expect(404)
+    .then((res) => {
+        expect(res.body.msg).toBe('Not Found')
+    })
+  })
+    
+})
```

Comment:
You are missing a test for a 400 where the article ID is of an invalid type. Should return a 400